src/core/diags/factory_diags.cpp

```diff
@@ -526,13 +535,8 @@ Error Diags::ProcessStop(uint8_t aArgsLength, char *aArgs[])
     Get<Radio>().SetPromiscuous(false);
     Get<Mac::SubMac>().SetRxOnWhenIdle(false);
 
-    Output("received packets: %d\r\nsent packets: %d\r\n"
-           "first received packet: rssi=%d, lqi=%d\r\n"
-           "last received packet: rssi=%d, lqi=%d\r\n"
-           "\nstop diagnostics mode\r\n",
-           static_cast<int>(mStats.mReceivedPackets), static_cast<int>(mStats.mSentPackets),
-           static_cast<int>(mStats.mFirstRssi), static_cast<int>(mStats.mFirstLqi), static_cast<int>(mStats.mLastRssi),
-           static_cast<int>(mStats.mLastLqi));
+    OutputStats();
+    Output("\nstop diagnostics mode\r\n");
```

Comment:
```suggestion
    Output("stop diagnostics mode\r\n");
```

Reply:
This '\n' just keep a line with original output format. Should we remove it?